learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+```
+layout: getting-started-with-ballerina-left-nav-pages
+title: Getting Started with Ballerina
+description: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+keywords: ballerina, programming language, ballerina packages, getting started
+permalink: /learn/getting-started-with-ballerina
+active: getting-started-with-ballerina
+intro: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+---
+```
+
+Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
```

Comment:
```suggestion
Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
```
We don't need to repeat this here again.

Reply:
Fixed. Thanks